cores/arduino/WInterrupts.c

```diff
@@ -250,8 +250,9 @@ void detachInterrupt(uint32_t pin)
  * External Interrupt Controller NVIC Interrupt Handler
  */
 #if defined(__SAMD51__)
-void InterruptHandler(uint32_t i)
+void InterruptHandler(uint32_t unused_i)
```

Comment:
something must be wrong with the existing code, but yeah, since nobody complain, we will just close an eye on this :D